### PR TITLE
Validate predict proxy chat

### PR DESCRIPTION
**Pull request by lferran**
### Description
[sc-14950]

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

**Review by drf7 — APPROVED** [review]

File: nucliadb/tests/nucliadb/integration/test_predict_proxy.py [hunk 1]
```diff
@@ -95,3 +95,45 @@ async def test_predict_proxy_returns_404_on_non_existing_kb(
...
+    stream_audit,
+):
+    kbid = standalone_knowledgebox
+    resp = await nucliadb_reader.post(
+        f"/kb/{kbid}/predict/chat",
+        json={
+            "question": "foo",
+            "query_context": ["foobar"],
+            "user_id": "foo",
+            "audit_metadata": {},
```
Comment (jotare) [inline nucliadb/tests/nucliadb/integration/test_predict_proxy.py:135]: Could you add some metadata, to test that it doesn't pass because it's empty

Comment (lferran) [reply nucliadb/tests/nucliadb/integration/test_predict_proxy.py:135]: I don't think it's needed. I just want to test with a valid payload of chat predict

**Review by jotare — APPROVED** [review]
